atlp-rwanda/elite-bn-frontend
# ft[update user profile]

What does this PR do?

- For authentication should be able to update his/her profile information

Description of Task completed

- User should be able to provide his or her profile info in in the provided field 

How should this be manually tested?

- git clone to https://github.com/atlp-rwanda/elite-bn-frontend.git
- checkout to ft-update-user-profile
- `npm install` 
- `npm start` to start the server or `npm run dev`
- Login by providing the email and password 
- And then after login user should be able t view his/her profile info
- On below profile info click on button for edit profile
- fill in the form the new field and click update to update user info
What is the relevant Trello story?

-https://trello.com/c/O4eKW9OE
